controllers/operands/kubevirt.go

```diff
@@ -270,6 +276,34 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1
 	return kv, nil
 }
 
+func hcoTuning2Kv(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+	var err error
+	if hc.Spec.TuningPolicy == hcov1beta1.HyperConvergedAnnotationTuningPolicy {
+		if annotation, ok := hc.Annotations[common.TuningPolicyAnnotationName]; ok {
+
+			var rates rateLimits
+			err := json.Unmarshal([]byte(annotation), &rates)
+			if err != nil {
```

Comment:
@jcanocan, maybe you also want to check that both fields are populated.

Reply:
You are right! I've missed that, I'm not sure if as it is, it's the best way to check it. Please, let me know.